Cargo.toml

```diff
@@ -2,162 +2,168 @@
-  "external-crates/move/move-binary-format",
-  "external-crates/move/move-binary-format/serializer-tests",
-  "external-crates/move/move-borrow-graph",
-  "external-crates/move/move-bytecode-verifier",
-  "external-crates/move/move-bytecode-verifier/bytecode-verifier-tests",
-  "external-crates/move/move-bytecode-verifier/fuzz",
-  "external-crates/move/move-bytecode-verifier/invalid-mutations",
-  "external-crates/move/move-bytecode-verifier/transactional-tests",
-  "external-crates/move/move-command-line-common",
-  "external-crates/move/move-compiler",
-  "external-crates/move/move-compiler/transactional-tests",
-  "external-crates/move/move-core/types",
-  "external-crates/move/move-ir-compiler",
-  "external-crates/move/move-ir-compiler/move-bytecode-source-map",
-  "external-crates/move/move-ir-compiler/move-ir-to-bytecode",
-  "external-crates/move/move-ir-compiler/move-ir-to-bytecode/syntax",
-  "external-crates/move/move-ir-compiler/transactional-tests",
-  "external-crates/move/move-ir/types",
-  "external-crates/move/move-model",
-  "external-crates/move/move-prover",
-  "external-crates/move/move-prover/boogie-backend",
-  "external-crates/move/move-prover/bytecode",
-  "external-crates/move/move-prover/interpreter",
-  "external-crates/move/move-prover/interpreter-testsuite",
-  "external-crates/move/move-prover/interpreter/crypto",
-  "external-crates/move/move-prover/move-abigen",
-  "external-crates/move/move-prover/move-docgen",
-  "external-crates/move/move-prover/move-errmapgen",
-  "external-crates/move/move-prover/test-utils",
-  "external-crates/move/move-stdlib",
-  "external-crates/move/move-symbol-pool",
-  "external-crates/move/move-vm/config",
-  "external-crates/move/move-vm/integration-tests",
-  "external-crates/move/move-vm/paranoid-tests",
-  "external-crates/move/move-vm/profiler",
-  "external-crates/move/move-vm/runtime",
-  "external-crates/move/move-vm/test-utils",
-  "external-crates/move/move-vm/transactional-tests",
-  "external-crates/move/move-vm/types",
-  "external-crates/move/testing-infra/enum-compat-util",
-  "external-crates/move/testing-infra/module-generation",
-  "external-crates/move/testing-infra/move-proc-macros",
-  "external-crates/move/testing-infra/test-generation",
-  "external-crates/move/testing-infra/transactional-test-runner",
-  "external-crates/move/tools/move-bytecode-utils",
-  "external-crates/move/tools/move-bytecode-viewer",
-  "external-crates/move/tools/move-cli",
-  "external-crates/move/tools/move-coverage",
-  "external-crates/move/tools/move-disassembler",
-  "external-crates/move/tools/move-explain",
-  "external-crates/move/tools/move-package",
-  "external-crates/move/tools/move-resource-viewer",
-  "external-crates/move/tools/move-unit-test",
-  "external-crates/move/tools/read-write-set",
-  "external-crates/move/tools/read-write-set/dynamic",
-  "external-crates/move/tools/read-write-set/types",
+    "external-crates/move-execution/suivm/move-bytecode-verifier",
```

Comment:
Hmm, it looks like my script likes to indent this by 4, and something else is indenting this by 2.  Let me know if this causes problems, and I'll change the script if so, but I'm pretty sure it is using the same library that `cargo` uses for manipulating toml files.

Reply:
I did not notice and I'll fix it.
I think the script may be a bit messed up if you put `-` or `_` in your version

Reply:
nothing to fix really....